test/app.js

```diff
@@ -8,21 +8,8 @@ const createAuthenticatedTestApp = h.createAuthenticatedTestApp;
 const createTestAppWithNetworkCB = h.createTestAppWithNetworkCB;
 const createTestAppWithOptions = h.createTestAppWithOptions;
 
-describe('Smoke test', () => {
-  it('unauthorised connection', () => {
-    const app = createTestApp();
-    return app.auth.genConnUri()
-      .then((resp) => {
-        should(resp.uri).is.not.undefined();
-        should(resp.uri).startWith('safe-auth:');
-      });
-  });
-
-  it('should build some authentication uri', () => {
-    const app = createTestApp();
-    return app.auth.genAuthUri({ _public: ['Read'] })
-        .then((resp) => should(resp.uri).startWith('safe-auth:'));
-  });
+describe('Smoke test', function testContainer() {
+  this.timeout(15000);
```

Comment:
In PR #171  I got rid of `function` keywords to use arrow functions everywhere. Please let's stick to arrow functions. 
In this case that you need to set a timeout, you can set it after the function body, i.e. `describe('Smoke test', () => {...}).timeout(15000);`